common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py

```diff
@@ -22,9 +22,19 @@ class Command(BaseCommand):
     of days are configurable from site configurations model though its default value is 3 days.
     """
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--site-id',
+            action='store',
+            dest='site_id',
+            type=int,
+            help='LMS site id, to get configuration from.'
+        )
+
     def handle(self, *args, **options):
-        ondemand_course_ids = CourseOverview.objects.filter(self_paced=True).values_list('id', flat=True)
+        site_id = options.get('site_id')
```

Comment:
We should handle if the site doesn't exist case.

Reply:
It is already handled, though I am updating docstring, as per discussion.